bimmer_connected/state.py

```diff
@@ -398,7 +398,6 @@ class Window(Lid):  # pylint: disable=too-few-public-methods
 
     A window can be a normal window of the car or the sun roof.
     """
-    pass
```

Comment:
An empty class does need `pass` -> https://docs.python.org/3.3/tutorial/classes.html#odds-and-ends

Reply:
The class has a docstring which sets the `__doc__` attribute and is therefore not empty. See also: https://github.com/PyCQA/pylint/issues/2208